theories/common.elpi

```diff
@@ -77,13 +77,13 @@ ground-uint {{ Number.UIntDecimal lp:D }} :- !, ground-decimal D.
 ground-uint {{ Number.UIntHexadecimal lp:D }} :- !, ground-hexadecimal D.
 
 pred reduction-pos i:term, o:term.
-reduction-pos I O :- coq.reduction.vm.norm I {{ positive }} O, ground-pos O.
+reduction-pos I O :- coq.reduction.cbv.norm I O, ground-pos O.
```

Comment:
Is that related to the port to HB? (same thing below and in lra.elpi and ring.elpi)

Reply:
Sorry, this is a workaround for a bug in jsCoq (or wacoq) which is required only for the MathComp school. I think now we can remove the second commit introducing this diff.

Reply:
But we should probably keep this commit (with the current hash) to avoid breakage of the MathComp installer for the school.

Reply:
I also guess that keeping the current commit has to be done in your fork (see [opam](https://www-sop.inria.fr/teams/marelle/MC-2022-installers/opam/packages/coq-mathcomp-algebra-tactics/coq-mathcomp-algebra-tactics.hierarchy-builder/opam) file). Could you put a tag like `MC-school-2022`?

Reply:
@pi8027 here it is: https://github.com/proux01/algebra-tactics/releases/tag/MC-school-2022

Reply:
Thanks!